package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "node_modules/.bin/eslint ./ || true",
     "test": "nyc --reporter=html --reporter=text mocha --require babel-core/register ./src/tests/ || true",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test"
+    "report-coverage": "nyc --reporter=lcov --reporter=text-lcov npm test",
+    "start": "nodemon ./bin/www --exec babel-node"
```

Comment:
You could add two scripts here. One to do the migrations, another to undo migrations. If you have any trouble executing this, check my repo https://github.com/oreoluwade/petproj/edit/develop/package.json for how I created the scripts.